src/common/doc/main_page.md

```diff
@@ -181,7 +181,7 @@
       <b>Author:</b> Vincent Rouvreau<br>
       <b>Introduced in:</b> GUDHI 2.2.0<br>
       <b>Copyright:</b> MIT [(GPL v3)](../../licensing/)<br>
-      <b>Includes:</b> [Miniball](https://people.inf.ethz.ch/gaertner/subdir/software/miniball.html)<br>
+      <b>Requires:</b> \ref cgal
```

Comment:
Actually, what part of CGAL do we use exactly? If it is only the kernel (no triangulation, kd-tree or other), then it is LGPL.